docker-credential-ecr-login.1

```diff
@@ -0,0 +1,91 @@
+.\" Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+.\"
+.\" Licensed under the Apache License, Version 2.0 (the
+.\" "License"). You may not use this file except in compliance
+.\" with the License. A copy of the License is located at
+.\"
+.\"      http://aws.amazon.com/apache2.0/
+.\"
+.\" or in the "license" file accompanying this file. This file is
+.\" distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+.\" CONDITIONS OF ANY KIND, either express or implied. See the
+.\" License for the specific language governing permissions and
+.\" limitations under the License.
+.TH DOCKER-CREDENTIAL-ECR-LOGIN 1 2018-10-29 AMAZON-WEB-SERVICES AMAZON-ELASTIC-CONTAINER-REGISTRY
```

Comment:
On a "standard" 80 column display, the header gets mashed together incomprehensibly:
`DOCKER-CREDENTIAL-ECR-LOAMAZON-ELASTIC-CONTAINER-REDOCKER-CREDENTIAL-ECR-LOGIN(1)`
Maybe you can use some short names and/or acronyms here?

Reply:
Does the updated version help?

Reply:
`DOCKER-CREDENTIAL-ECR-LOGIN(1)       AMAZON-ECR       DOCKER-CREDENTIAL-ECR-LOGIN(1)`

Looks much better, thanks.